.github/workflows/plugins-ci-redis.yml

```
@@ -0,0 +1,140 @@
+name: Plugin CI - Redis
+
+on:
+  workflow_call:
+    inputs:
+      auto-merge-exclude:
+        description: 'A comma separated list of packages that you do not want to be auto-merged.'
+        required: false
+        default: 'fastify'
+        type: string
+      license-check:
+        description: 'Check licenses'
+        required: false
+        type: boolean
+        default: false
+      license-check-allowed-additional:
+        description: 'Additional allowed licenses (separator ;)'
```

```suggestion
        description: 'A semicolon seperate list of additional licenses to allow.'
```

So it's in line with the others. :)